app/appshell/app.js

```diff
@@ -210,8 +211,8 @@ app.getRemoteDebuggingPort = function () {
 };
 
 app.getUserDocumentsDirectory = function () {
-    // TODO: implement
-    throw new Error("app.getUserDocumentsDirectory not implemented");
+    var _path = path.resolve(electronApp.getPath("home"), "Documents");
```

Comment:
what does Brackets even use this for? The concept of Documents vs. projects is pretty different, imo.